apps/civicsignalblog/src/payload/collections/CivicSignalBlog/Pages.js

```diff
@@ -0,0 +1,31 @@
+import canRead from "../../access/applications/researchBlog";
```

Comment:
Can't we get aliases to work in Payload? Not necessary in this PR but these relative paths are hard to keep up with.

Reply:
Yeah that was the plan (I didn't do it in this PR to reduce the number of files changed)... working with relative paths has been a pain